lib/pact/command/cont_command.ex

```diff
@@ -74,7 +74,7 @@ defmodule Kadena.Pact.ContCommand do
     :step,
     :proof,
     :rollback,
-    :signers,
+    signers: [],
```

Comment:
@Odraxs @EdwinGuayacan  `@type signers` indicates that this attribute must be a `SignerList.t()`, but we're setting by default an empty list, do you think this is right? 🤔 Same for ExecCommand...